webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/esbuild.js

```diff
@@ -0,0 +1,45 @@
+// Load the go wasm.
+require("./go-wasm");
+
+class EsBuild extends Go {
+  constructor() {
+    super();
+    this.wasm = undefined;
+    this.workDir = "/app";
+    this.user = "esbuild";
+    this.homeDir = "/home/esbuild";
```

Comment:
I'd suggest we used the patched "os" module here to get the mock user and home values

Reply:
forgot this was here. Not needed for this. I used these with the node version of the go wasm exec for esbuild but that went nowhere. I am removing it.